### PR TITLE
Project Loved: February Week 1 Newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Lots of links in this one, will need some thorough checking.

### CODE REVIEW

File: news/2018-02-05-project-loved-february-week-1.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+Since it is a lower star difficulty than the majority of the maps featured this week, this is a highly accessible mapset for players of every skill level.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/guitar-vs-piano.jpg)](https://osu.ppy.sh/community/forums/topics/699481)
+
+[Shiro](https://osu.ppy.sh/u/113005)'s latest masterpiece, GvP gained quick popularity as an equally difficult jump and stream map. Shiro makes great use of various patterns throughout a very repetitive song including a guitar slider and an outro treble clef. 
+
+I wish you Good Luck in G Major.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/grenade.jpg)](https://osu.ppy.sh/s/563413)
```
Comment (WalterToro) [inline news/2018-02-05-project-loved-february-week-1.md]: Should be https://osu.ppy.sh/community/forums/topics/699482

File: news/2018-02-05-project-loved-february-week-1.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+Returning for another chance at Loved after the initial rounds of voting, Grenade by [Nanashima Asami](https://osu.ppy.sh/community/forums/topics/699482) is a very well constructed tech map that has been featured in multiple tournaments.
```
Comment (WalterToro) [inline news/2018-02-05-project-loved-february-week-1.md]: Should be https://osu.ppy.sh/u/7370134

**Comment by Ephemeralis** [comment]
Fixed.

File: news/2018-02-05-project-loved-february-week-1.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/foolish-hero.jpg)](https://osu.ppy.sh/community/forums/topics/699478)
+
+[Cassu2](https://osu.ppy.sh/u/2587961)'s Foolish Hero bundles lapix's Hi-Tech style with some interesting high-tech slider patterns themselves. Overall, this proves to be an incredibly enjoyable and popular addition to the Loved vote this week.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/mazare-party.jpg)](https://osu.ppy.sh/community/forums/topics/699479)
+
+Mazare Party as a song is a collaboration between 3 popular Japanese vocalists, and likewise the map is a collaboration between 3 popular mappers! 
+
+[ProfessionalBox](https://osu.ppy.sh/u/3250792), [Asahina Momoko](https://osu.ppy.sh/u/3650145), and [Nathan](https://osu.ppy.sh/u/4785223) join forces in a fantastic collab to a fantastic song, with a few lower difficulties by [toybot](https://osu.ppy.sh/u/2848604) and [NeilPerry](https://osu.ppy.sh/u/2848604) to fill out the set.
```
Comment (WalterToro) [inline news/2018-02-05-project-loved-february-week-1.md]: I actually missed this one, NeilPerry should be https://osu.ppy.sh/u/841391 (links to toybot profile)

File: news/2018-02-05-project-loved-february-week-1.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+Souzou Forest is one of [17VA](https://osu.ppy.sh/u/1895984)'s most popular charts, it is a fun LN chart which is decently fast but also fairly simplistic stair based patterning, which makes it both easy to hit and hard to get good accuracy on. 
+
+It has been featured in the MWC 7K 2016 Quarterfinals and has retained its popularity to this day.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!mania/the-scales-of-struggle.jpg)](https://osu.ppy.sh/community/forums/topics/699503)
+
+As the song title suggests, The Scales of Struggle scales the amount of struggle with each step up the difficulty ladder. Ranging from easy peasy to outright warfare, each difficulty tests the waters of speedy technicality for the whole playerbase.
+
+A great collaboration between veteran charter Fullerene- and veteran speedster Staiain, which we're sure you'll enjoy.
```
Comment (WalterToro) [inline news/2018-02-05-project-loved-february-week-1.md]: Fullerene- (https://osu.ppy.sh/u/2531335) and Staiain (https://osu.ppy.sh/u/86188) profiles should be added

File: news/2018-02-05-project-loved-february-week-1.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!mania/peace-breaker.jpg)](https://osu.ppy.sh/community/forums/topics/699504)
```
Comment (WalterToro) [inline news/2018-02-05-project-loved-february-week-1.md]: This image is broken because there is a space in the filename (the one uploaded)

**Comment by Ephemeralis** [comment]
Excellent eye. I was project blind for half of these, so this is a big help.

**Pull request merged**